moped-api/claims.py

```diff
@@ -253,4 +253,4 @@ def decrypt(fernet_key: str, content: str) -> str:
     :return str: The decrypted string
     """
     cipher_suite = Fernet(fernet_key)
-    return cipher_suite.decrypt(content.encode()).decode()
+    return cipher_suite.decrypt(content.encode()).decode()
```

Comment:
Not a problem here either, but the python linter is going to bark about not having an empty line at the end of the file

Reply:
Good call! I was battling my formatter because it was trying to make a bunch of changes. I missed that one so I'll update before merge. 🙏